CheckThePossibilityBuiltThreeAngle.java

```diff
@@ -0,0 +1,19 @@
+import java.util.Scanner;
+public class CheckThePossibilityBuiltThreeAngle {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Input the lengths of threeangle");
+        int Side1 = sc.nextInt();
+        int Side2 = sc.nextInt();
+        int Side3 = sc.nextInt();
+        if (Side1 <= 0 || Side2 <= 0 || Side3 <= 0) {
+            System.exit(1);
+        } else {
+            if ((Side1 + Side2) < Side3 || (Side1 + Side3) < Side2 || (Side2 + Side3) < Side1) {
```

Comment:
The condition isn't fully correct.

Reply:
Understood, the conditions should be <= .